### DISCUSSION TITLE
Should we keep the capitalization of names?

### DISCUSSION THREAD

**Discussion by 5axes** [5axes/NameIt#2] [Ideas]
Currently the plugin converts all the names in capital letters. This is normal because I think that the letters are easier to print in this case (example of i and j which in capital letters has no dot) .

But some feedback are expected on this point. Could be in a futur release integrated as an option in the settings of the plugin.

**Comment by ImagineerNL** [comment]
In my opinion, for sake of clarity, capitals should be the way to go for the reasons you describe. 
However; it all depends on the fonts you're using. A Serif font (like times new roman) with thinner lines and pointy edges can create more trouble than a uncapotalized letter. 

A sans serif font like Open Sans which is built up from a lot of curves Seems to be a good one.

for lasercutting and such, a shf, or single line font is viable; maybe this is a safe starting point as well. 

Bigger issue besides caps yes/no might be aloowing the use of special characters (ë à æ ç) and non-western texts. Both non-western (right to left?) and special characters can have the same issues regarding 'small details'

Also... it's for 'naming'. Its not like someone should use it to write an essay on the printbed.

**Comment by 5axes** [comment]
We can also decide that these characters will be output as basic one ( It's not the case right now) IE

- Ã-> A
- Ê-> E
- Î-> I
- Ñ->N

Etc.
